.github/workflows/weekly.yml

```diff
@@ -34,6 +34,11 @@ jobs:
           branch: main
         }, {
           flink: 1.18-SNAPSHOT,
+          jdk: '8, 11, 17',
+          branch: main
+        }, {
+          flink: 1.19-SNAPSHOT,
+          jdk: '8, 11, 17, 21',
```

Comment:
Why is is this a part of the PR?
It is already in main

I guess you need to rebase

Reply:
yes there was a merge conflict I will sort this out